packages/@aws-cdk/aws-amplify/lib/app.ts

```diff
@@ -0,0 +1,347 @@
+import { IRole, LazyRole, ServicePrincipal } from '@aws-cdk/aws-iam';
+import { Construct, IResolvable, IResolveContext, IResource, Resource, SecretValue, Tag } from '@aws-cdk/core';
+import { CfnApp } from './amplify.generated';
+import { Branch, BranchBaseProps } from './branch';
+import { BuildSpec } from './buildspec';
+import { Domain, DomainBaseProps } from './domain';
+import { BasicAuthConfig, BasicAuthResolver, EnvironmentVariable, EnvironmentVariablesResolver } from './shared';
+
+/**
+ * App
```

Comment:
Missing docs here and everywhere. Don't leave empty docs. Ideally we like to copy & paste the official AWS docs throughout the API.